database/pgsql/queries.go

```diff
@@ -141,6 +141,9 @@ const (
 	searchVulnerabilityForUpdate          = ` FOR UPDATE OF v`
 	searchVulnerabilityByNamespaceAndName = ` WHERE n.name = $1 AND v.name = $2 AND v.deleted_at IS NULL`
 	searchVulnerabilityByID               = ` WHERE v.id = $1`
+	searchVulnerabilityByNamespace        = ` WHERE n.name = $1 AND v.deleted_at IS NULL
+						  ORDER BY v.name
+						  LIMIT $2 offset $3`
```

Comment:
For performances, we try to avoid using `OFFSET` but instead use `WHERE id >= $3 ORDER BY l.ID LIMIT $2`. And then for security purpose, we encrypt the id at the API level. See how notifications work.